source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -1,121 +1,155 @@
-
-#. **Add Rpms and create content**
-run::
-      # sudo swupd-add-pkg <pkg_name> <bundle_name>
-
-   This automatically creates the <bundle_name> bundle if it does not exist,
-   adds the RPM name to it, and generates all the mixer content and Manifests
-   needed.
-
-#. **Migrate to local setup**. Now that the content exists, the easiest way to have swupd
-   become aware of it and use it, is to use the new :command:``--migrate`` flag for an update.
-run::
-   # sudo swupd update --migrate
-
-   This will update your system to the mix ready state using the new Manifest.MoM.
-
-#. **Add your bundle to your system** To add your bundle, simply run::
-
-   # sudo swupd bundle-add <my new bundle>
-
-
-
-Revert back to upstream
--------------------------
-
-You may wish to go back to official upstream with no user content. This can
-easily be done by calling :command:``swupd verify --fix --picky --force -m
-<upstreamversion> -C /usr/share/clear/update-ca/Swupd_Root.pem``, which will
-verify fix the system back to the upstream version you are augmenting with
-local content. From here on regular swupd commands will only reach out to
-the upstream server, since /usr/share/mix will no longer exist.
+*************
+
+To use the integrated swupd mixer, you need a recent image of
+|CLOSIA| with the mixer bundle installed. If you do not have it already,
+add it with the :command:`swupd bundle-add` command as shown below:
+
+.. code-block:: bash
+
+   sudo swupd bundle-add mixer
+
+
+Using the enhanced swupd mixer
+******************************
+
+Perform the following steps:
+
+#. Create a folder :file:`/usr/share/mix/rpms` for the local update content
+   and copy the desired RPMs into the folder:
+
+   .. note::
+
+     The RPMs must be formatted and built for |CL|.
+
+   .. code-block:: bash
+
+      sudo cp <RPM1 RPMn> /usr/share/mix/rpms/
+
+#. Add RPMs and create content.
+
+   .. code-block:: bash
+
+      sudo swupd-add-pkg <RPM1 RPMn> <bundle_name>
+
+   The :file:`swupd-add-pkg` script creates the :command:`<bundle_name>`
+   bundle if it does not exist, adds the RPMs to it, and generates the
+   required mixer content and manifests.
+
+#. Use the new :command:`--migrate` flag to let :command:`swupd` know that
+   local content exists.
+
+   .. code-block:: bash
+
+      sudo swupd update --migrate
+
+   When complete, your system is in the mix-ready state using the new
+   Manifest.MoM.
+
+#. Add your bundle to your system:
+
+   .. code-block:: bash
+
+      sudo swupd bundle-add <bundle_name>
+
+
+Reverting to the upstream version
+*********************************
+
+You can return your OS to an official upstream version with no user-added
+content using the following command:
+
+   .. code-block:: bash
+
+      sudo swupd verify --fix --picky --force -m <upstreamversion> -C
+      /usr/share/clear/update-ca/Swupd_Root.pem
+
+   .. note::
+
+      The options used in the example delete the :file:`/usr/share/mix` folder.
```

Comment:
> +      The options used in the example delete the :file:`/usr/share/mix` folder.
".. and all other extraneous data on your system that is not part of <upstreamversion>" definitely should be added here for transparency.